sponsor/gs3-crowdfunding.md

```diff
@@ -214,12 +236,12 @@ MapStore, GeoNode and Cloud Native GeoServer have been prioritized by initial sp
 
 #### Q: OGCAPI Features is mentioned, is this included in the GeoServer 3 crowdfunding activity?
 
-Camptocamp is hosting a codesprint in early November 2024 on this topic and we anticipate having OGCAPI Features [ready in time](https://github.com/geoserver/geoserver/wiki/GSIP-230) for GeoServer 3.
+Camptocamp hosted a codesprint in early November 2024 on this topic and OGCAPI Features was [ready in time](https://github.com/geoserver/geoserver/wiki/GSIP-230) for GeoServer 2.27.0.
 
 The transition from OGC Open Web Services (WMS, WFS, WCS) to OGCAPI services will take place over the next several years. Adjusting the GeoServer user interface with this transition is included as part of the GeoServer 3 project plan.
 
 #### Q: Will a specific community module be included?
 
-Check the project plan to see if the community module you are interested will be available in GeoServer 3.
+Check the [Project Plan](https://docs.google.com/document/d/1EmI1kDsqeoxB9GANiiaZy56RY0pWbl6GqD4fq8EJ4oo/edit?tab=t.0) to see if the community module you are interested will be available in GeoServer 3.
```

Comment:
Add `in` after interested. 

Also `will`? Maybe `is compatible with`, or `needs your help to be included in`?